<launch>
  <node pkg="obst_avoid" name="img_prc" type="img_prc" respawn="true" output="screen"/>
</launch>
